frontend/src/components/BattleLogItem.tsx

```diff
@@ -14,14 +14,14 @@ const BattleLogItem: React.FunctionComponent<Props> = ({
   units,
   enemyUnits,
 }: Props) => {
-  const actionVerb = action.action === "kill" ? " killed" : " damaged";
+  const actionVerb = action.action === "kill" ? " kills" : " attacks";
   const allUnits = [...units, ...enemyUnits];
   const who = allUnits.find(unit => unit.id === action.who) || units[0];
   const whom = allUnits.find(unit => unit.id === action.whom) || units[0];
 
   const positiveNews = units.indexOf(who) !== -1;
-  const whoDescription = positiveNews ? "You " : "Enemy ";
-  const whomDescription = !positiveNews ? " you " : " enemy ";
+  const whoDescription = positiveNews ? "Your " : "Enemy ";
```

Comment:
If we have "your" shouldn't second one be "enemy's"

Reply:
Not sure. I tried to make the battle log message shorter so that it fits in one line on most screens. I think you can say: "your unit", and "enemy unit". I think that's gramatically correct, is it not?
And in our case... well, the "unit" word is substituted by its icon.

Reply:
Okey, sounds correct